libs/knowledge-store/ragstack_knowledge_store/cassandra.py

```diff
@@ -245,28 +248,20 @@ def embeddings(self) -> Optional[Embeddings]:
     def _concurrent_queries(self) -> ConcurrentQueries:
         return ConcurrentQueries(self._session, concurrency=self._concurrency)
 
-    # TODO: async
-    def add_texts(
+    def add_nodes(
         self,
-        texts: Iterable[str],
-        metadatas: Optional[Iterable[Dict[str, Any]]] = None,
+        nodes: Iterable[Node] = None,
+        links: Iterable[Tuple[str, str]] = None,
```

Comment:
Links is no longer part of the signature.

Reply:
done